docker-compose.yaml

```diff
@@ -0,0 +1,69 @@
+version: "3.8"
+
+services:
+  postgres:
+    image: timescale/timescaledb:0.9.1-pg10
```

Comment:
timescaledb was used for feedback feature (which are not working now because we dropped taskqueue support)  
I suggest to replace this with plain postgres and drop timescaledb-specific stuff.  
But maybe as part of separate pr